packages/sanes-chrome-extension/src/routes/signup/test/travelToSignup.ts

```diff
@@ -16,3 +19,9 @@ export const travelToSignup = async (store: Store): Promise<React.Component> =>
 
   return dom;
 };
+
+export const travelToSignupNewAccountStep = async (page: Page): Promise<void> => {
+  await page.click('button:nth-of-type(2)');
+
+  await findRenderedE2EComponentWithId(page, FIRST_STEP_SIGNUP_ROUTE);
```

Comment:
I had in mind to use `https://github.com/GoogleChrome/puppeteer/blob/v1.15.0/docs/api.md#pagewaitfornavigationoptions` for checking navigation, but seeing we have used ids in DOM tests I really like your solution.